install/kubernetes/helm/istio/charts/prometheus/templates/deployment.yaml

```diff
@@ -45,6 +45,7 @@ spec:
           args:
             - '--storage.tsdb.retention={{ .Values.retention }}'
             - '--config.file=/etc/prometheus/prometheus.yml'
+            - '--storage.tsdb.path={{ .Values.persistentVolume.mountPath }}'
```

Comment:
Shouldn't this be wrapped with `if .Values.persistentVolume.enabled` ?

Reply:
Good point. This has been fixed.